web-server/django_web_server/website/server.py

```diff
@@ -130,16 +129,20 @@ def getTasks(self):
                     c_id = task['_id']['client_id']
                     task_t = task['task']['name']
                     task_i = task['task']['data']
+                    t_date = task['task']['date']
+                    t_time = task['task']['time']
                     t_status = 'Pending'
-                    self.__saveTask(t_id, c_id, task_t, task_i, t_status)
+                    self.__saveTask(t_id, c_id, task_t, task_i, t_status, t_date, t_time)
             for task in sent_tasks:
                 t_id = task['_id']['task_id'] + task['_id']['client_id']
                 if not (SentTask.objects.filter(task_id=t_id).exists()):
                     c_id = task['_id']['client_id']
                     task_t = task['task']['name']
                     task_i = task['task']['data']
+                    t_date = task['task']['date']
```

Comment:
The data sent from the control server no longer looks like this, therefore this code does not work any more. Maybe we should combine this pull request with another branch where I have fixed this issue?